setup/install/providers/index.md

```diff
@@ -24,7 +24,6 @@ Add as many of the following providers as you need. When you're done, return to
 
 * [App Engine](/setup/install/providers/appengine/)
 * [Amazon Web Services](/setup/install/providers/aws/)
-* [Amazon Web Services - ECS](/setup/install/providers/ecs/)
```

Comment:
It might make sense to have both EC2 and ECS listed since they provide different functionalities ( i.e, App Engine vs. Google Compute Engine ) is an example

Reply:
I consolidated EC2 and ECS under the breadcrumb of Amazon Web Services. My thought is to provide a uniform experience and enhance ECS related setup as well in the same way as done for EC2

![screen shot 2018-06-14 at 1 52 26 pm](https://user-images.githubusercontent.com/20847838/41437773-3ea77520-6fda-11e8-81d8-678c1486df38.png)